pyppeteer/execution_context.py

```diff
@@ -29,209 +28,107 @@
 class ExecutionContext(object):
     """Execution Context class."""
 
-    def __init__(self, client: CDPSession, contextPayload: Dict,
-                 objectHandleFactory: Any, frame: 'Frame' = None) -> None:
+    def __init__(
+            self,
```

Comment:
I think we should adopt an auto-formatter for this project, just for the sake of consistency and readability. Whatever rules we come up with I don't really care, but I do care that it's consistent.

Reply:
Sure, we can give `black` a spin though maybe after 2.1.1 migration. I'm not sure how well would it work with our string heavy workflow - black in my experience tend to make it quite messy.

Reply:
[FWIW, here's what black looks like](https://black.now.sh/?version=stable&state=_Td6WFoAAATm1rRGAgAhARYAAAB0L-Wj4BQ3BoddAD2IimZxl1N_Wg0_QLDSluJRcxyy_agGIjCIe-3gOSwQBIqqsjr3JZ5rOCMEEh8drQYkm49tf_t959qOMZ4v3nMNfk7zsl3FCnTVz7oPkzUUdTovpl2qijjewo-Jp1o3NBPs7GzaRbq6-6Yy6zjFPPJXVFTdNpUFu4uZdr-OUeLI9E9JX6u9cVVO12Eoe7hfRQ-ePXnbAAcodhcSux3CspD4USicwW_GGamdkMj7FmpSdeUe2UnuZnlxUZWOc3fQ7U4HLSN5N-wu5qbJi8OKx9skVt8bhTi3Hbz9_qzwZOazwahYT1S60BVDsaAcx__tNp2wgR4xap58qU3NAaaAhTJoEcwF1UcboENz74wKtiYhoPpTzlzsovIHyidBGAo39dmzVHLPksKJre7w3J0Y_lDTWejBIrfwQigsPBJ8Owjtp2zufoUMOlE08JHmBaOdrFivoOLWtha3Qy_h3OcRxCO5KJl0FXPUHoVKGVJvnrITmY11NVQz17Y_bskkVTdhOPv7j4BPzxhgUiQwPX-YO69_SDtAD_xp5n2B5Oz9ZIo3-_jVSdXxPB7jRNnjL95FVo6ATCgB9PO70J0Rmx6TXYU4qFRv0CjZCwqmYiezDzMyGpIc2cj0Qo0p1Wlkv9-3M7aaLEF15i9VODq6DxVtscpiM-7qbfBO7p4dBP95a4X8C2NfNS90vVGOjNMjBQTV-vQFgFqJYSrSnR6EchHm1ImhFBt_Z2eeVbI1Wu2LoLyc-eCOY14iksLOTSvWioqOL8RKPA1lGDYOiwOY8AphGMBmo4UXY3FA9LSrbBrqZzwjE7Borjekz1fjW-QOcND8dXr1MlUQ88SOtyDBRNlvrIxLk84qKCxj61lWL5ZmaUYFxwO-v2hMEFORvK-omMzpktCU99eUTSCCNXPUHjvGYXKRF3GkDq9v-mwH2LMJryFjW0T1yBzalE1x4p-ZvHrZ0M3KUkcb7Jek7zyxuSg6lTyxbXwLSif2LYjkmcfWlxtwbcA6hMLxnrziKfUakZgRRuvQIUqOePK4GAnzXt5m1Y4h4SCA6thVmRTQe3oxSksY8Fmsoy7z5pJWIS2CIu4NXEjyaenvOInv4SgddrSGQzX7u0XP8ko2xlqrHsxn_bm4sK4UF5h7pKjH1P6Ql5KcdWBVB7InkZkasipKSBt6Ue5JEWRenwsVwykFwShj0Yxm-7SkHMvLU4a39EeKUpjImNDn3I8s7zdmgBBPmX9Ixyy8wWymOMbF-7bbB2cPGztPzMKKhBF6NiSsVVCibbxqj3qPsI9ad4NYH7i5UTJWfkDTrMlDh4Ktvuw5b0G3NnDptilOMd2Tg14MDh32q2pMNUNNaqZfKOr8EZmh3P9cqrQPZJjuv8y4Xqi0neooNyWeGL-q9U75w5OGmwPwcbIoQtd3l3n-G0VZRRJmgbt_bBnq-sH9QB75K5b7UUCVeDOU_laIobey0WTPYb6sXG4X_wHPPl3Tamie_q3Dqomc-IrF7smQJ-4x9JY6fhOE_O1hFcWrwjeiF01aexXEo_JKRbrWb6LWjnv_46Osv08J9lm8h57DXRQ1FR4eaOYT8jDWV5pfqq93T5mqOZ8UF25oc2N7P-1HOSBJac0mYpXnGomnsMJuPTmXv_ZtrTOMCBIuCzRrlz9egKgicvnB4MR06u9-1c1i_gd9SmtLGYmQPXzLN5J_EgEG5JogvWoN75l8LoJXQcbhR3W_OFHyHsq3ZzaqxufrtuPaKyGE4ANDjPV4rIghUybMEdCjQv4u8jUpBFH1SaMoV4i-lU_98OGhncx4rMNwPWInzX5w1SzGwpyyryYyRgw-tYAx2vG5KHfOpdDf_drK2OE4y_TykWvJ75rDcbMspKlMDKaFUBNq4lH7mV1D8p2r_bSr1_5ywyILyZXG9oy-XUW8lNt9N8pF3SX8KSNJ0Bnkda5MZzRIcFvM95vQ0QPV93lJHt0SI06OcQwICmwe-__iVVhxRFuj_B_Fwx3yCcOk9gJ4OqabZt0Khv5AgR1UG_ulU94w5slPUKqyDDUZOhM7FzoMUsCFVlU0xuikOVdabMXGHvElqLcmW80UCOju0gKhNkKPbn3Jafc5vwNTwp7UPHcdnYr56kYfVHu_ZCWAHe4ApKKw3mFFNdM-a955VNMHPJQR1ZUsyQ-OXB55zs63lIpOtZm6ILJrN-fWGQpD63ilhMCOZ4xIsxI5ed5UL6iFp3bP6iTavorIwOS-mgAAMdhJUbn2efQAAaMNuCgAAAEC1UqxxGf7AgAAAAAEWVo=)